### PR TITLE
Sparkline chart changes

### PR DESCRIPTION
**Pull request by wojta**
Trying to replicate the same approach as in #209 for the sparkline chart training.

### CODE REVIEW

File: react-charts/sparkline-chart/step1.md [hunk 1]
```diff
@@ -1,10 +1,30 @@
...
-
-<img src="sparkline-chart/assets/final.png" alt="Completed chart component" style="box-shadow: rgba(3, 3, 3, 0.2) 0px 1.25px 2.5px 0px;" />
+Begin by adding a CSS selector to define the chart size.
+
+Katacoda is setting up a new React application. Begin coding once the server
+starts and "Welcome to PatternFly" is visible on the lower pane.
+
+1) <strong>Navigate to the `src` folder and open `src/app.css`{{open}}</strong>
+
+2) <strong>Add CSS to the overall chart size and label layout.</strong>
```
Comment (christiemolloy) [inline react-charts/sparkline-chart/step1.md]: @dlabrecq could you add a sentence here for what the label layout refers to?

File: react-charts/sparkline-chart/step4.md [hunk 2]
```diff
@@ -1,36 +1,23 @@
-Now we'll add tooltips to the sparkline chart...
+Add a label to the sparkline chart.
 
-## Task
+1) <strong>Make sure `App.js` is still open.</strong>
 
-1) Make sure the App.js file is still open
-
-2) Locate the code that looks like the following:
+2) <strong>Locate the code that looks like this:</strong>
```
Comment (christiemolloy) [inline react-charts/sparkline-chart/step4.md]:
```suggestion
2) <strong>Locate the code for the `chart-container` wrapper.</strong>
```

File: react-charts/sparkline-chart/step4.md [hunk 3]
```diff
@@ -1,36 +1,23 @@
...
-  height={100}
-  width={400}
-&gt;
+&lt;div className=&quot;chart-container&quot;&gt;
+  ...
+&lt;/div&gt;
 </pre>
 
-3) Add the following property to that section:
+3) Add the following code inside the `<div>`
```
Comment (christiemolloy) [inline react-charts/sparkline-chart/step4.md]:
```suggestion
3) Add the `ChartContainer` component outside of the wrapper identified in step 2.
```

(@dlabrecq can you confirm that this is right)

Comment (christiemolloy) [reply react-charts/sparkline-chart/step4.md]: And then add a step for adding the Chart Label inside of the Chart Container. also explain why you need a chart container component

File: react-charts/sparkline-chart/step1-answer.md [hunk 4]
```diff
@@ -1,4 +1,5 @@
-Please copy the following code into the app.css file, replacing all of the content there:
+Please copy the following code into the app.css file, replacing all of the
```
Comment (janwright73) [inline react-charts/sparkline-chart/step1-answer.md]: backticks needed. Copy the following code into the `app.css` file, replacing all the content.

Comment (gheake2) [inline react-charts/sparkline-chart/step1-answer.md]:
```suggestion
Please copy the following code into the `app.css file`, replacing all of the
```

File: react-charts/sparkline-chart/step1.md (hunk 1 above)
Comment (dlabrecq) [reply react-charts/sparkline-chart/step1.md]: axis labels, legend labels, etc.

**Comment by dlabrecq** [comment]
Would it be too much trouble to ask that these PR reviews provide a link to the working tutorial? 

I encountered a few broken steps in previous reviews that could only be found by running the tutorial. It would also be much easier to review the numerous changes in context.

File: react-charts/sparkline-chart/step3.md [hunk 5]
```diff
@@ -1,56 +1,52 @@
...
+3) <strong>Add a `containerComponent` property to the `ChartGroup` component.</strong>
+
+The `containerComponent` property takes a component instance, which will be used
+to create a container element for standalone charts.
+Inside the `containerComponent` is the `ChartVoronoiContainer` component, which
+adds the ability to associate a mouse position with the data point(s) closest to
+it. This is useful for adding a tooltip.
+
+<pre class="file" data-target="clipboard">
+containerComponent={
```
Comment (christiemolloy) [inline react-charts/sparkline-chart/step3.md]: This should just be:
containerComponent={
  <ChartVoronoiContainer
  />
}

File: react-charts/sparkline-chart/step3.md [hunk 6]
```diff
@@ -1,56 +1,52 @@
...
+It should look like this:
+
+<pre class="file" data-target="clipboard">
+containerComponent={
+  &lt;ChartVoronoiContainer
+    constrainToVisibleArea
+  /&gt;
+}
+</pre>
+
```
Comment (christiemolloy) [inline react-charts/sparkline-chart/step3.md]: Add step 5 for the labels, to that its

containerComponent={
  <ChartVoronoiContainer
    constrainToVisibleArea
    labels={({ datum }) => `${datum.name}: ${datum.y}`}
  />
}

look at area-chart: step 4, no.5

File: react-charts/sparkline-chart/step4.md (hunk 3 above)
Comment (christiemolloy) [reply react-charts/sparkline-chart/step4.md]: i think its safe to commit the suggestion above ^ @wojta

Comment (wojta) [reply react-charts/sparkline-chart/step4.md]: >  Add the `ChartContainer` component outside of the wrapper identified in step 2.
outside?

Comment (christiemolloy) [reply react-charts/sparkline-chart/step4.md]: Yep can confirm it sits below the identified section:
This is from the original Sparkline

Screen Shot 2019-12-10 at 11 43 13 AM

**Pull request merged**